doc/requirements.txt

```diff
@@ -21,3 +21,4 @@ tensornetwork==0.3
 toml
 torch==1.9.0+cpu
 torchvision==0.10.0+cpu
+black>=21
```

Comment:
@Qottmann it's a good idea to include this! Better though to place this in the [`requirements.txt`](https://github.com/PennyLaneAI/pennylane/blob/709221385b456d91de23b99652700de9b0027f8d/requirements.txt) file, as this one seems to break the documentation from building 🤔 I will remove it for now, just so that the documentation build correctly.
```suggestion
```

Reply:
Ah right! I added it to this requirement.txt. The remaining error: do I understand it right, that it is complaining that the case that argnum and tensor_list are given as integers, is not tested? I can modify the test to include those tests then!